library/Icingadb/Widget/Detail/ObjectDetail.php

```diff
@@ -70,12 +71,14 @@ protected function createActions()
 
         foreach ($this->compatObject->getActionUrls() as $i => $url) {
             $navigation->addItem(
-                sprintf(t('Action %d'), $i + 1),
+                (string) new Icon('forward', [
+                    'title' => t('Link opens in new window')
+                ]) . (string) new Text(" " . $url),
                 [
-                    'renderer' => [
+                    'renderer' => array(
```

Comment:
Why did you change this back to the legacy syntax? We prefer brackets instead of this so please revert this.